Kinvey-Xamarin/Store/PushRequest.cs

```diff
@@ -57,14 +57,16 @@ public PushRequest(AbstractClient client, string collection, ICache<T> cache, IS
 					catch (Exception e)
 					{
 						//Do nothing for now
-						response.addError(new KinveyJsonError());	//TODO
+						response.AddKinveyException(new KinveyException(EnumErrorCategory.ERROR_DATASTORE_NETWORK,
+						                                                EnumErrorCode.ERROR_JSON_RESPONSE,
+						                                                ""));  // TODO provide correct exception
 					}
 				}
 
 				await Task.WhenAll(tasks.ToArray());
 				foreach (var t in tasks)
 				{
-					response.Count += t.Result;
+					response.PushCount += t.Result;
```

Comment:
@vinaygahlawat you should also set `response.Entities` here. The entities are in each network response.